clang/lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -628,6 +628,10 @@ static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
     return IsStructurallyEquivalent(Context, Arg1.getAsExpr(),
                                     Arg2.getAsExpr());
 
+  case TemplateArgument::StructuralValue:
+    // FIXME: Do we need to customize the comparison?
```

Comment:
That looks correct to me, do you think we still need the fixme?

Reply:
I agree that it seems like it should work. Dropped the comment.